folder-selection/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "folder-selection",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "deploy": "npm run build && npx @storyblok/field-plugin-cli@beta deploy"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.14.7",
+    "@storyblok/field-plugin": "0.0.1-beta.4",
```

Comment:
could you upgrade this to `1.0.1`? there is one breaking change. There is no longer `FieldPluginProvider`. I've left more explanation on App.tsx

Reply:
Alright! 
Sorry, I didn't know that. I might have missed it.